src/main/java/org/isf/dicom/service/DicomIoOperations.java

```diff
@@ -48,14 +48,14 @@ public class DicomIoOperations
 	private DicomIoOperationRepository repository;
 	
 	/**
-	 * Load a list of id file for series
+	 * Load a list of file iss for a series.
```

Comment:
Like this?
```suggestion
	 * Load a list of file id for a series.
```

Reply:
yes and adding the `s` to `serie`;   `series` is both singular and plural :smile: